ygot/testdata/exschemapath/exschemapath.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package exschemapath;
+
+option go_package = "github.com/openconfig/ygot/ygot/testdata/exschemapath";
+
+import "github.com/openconfig/ygot/proto/ywrapper/ywrapper.proto";
+import "github.com/openconfig/ygot/proto/yext/yext.proto";
+
+message Root {
```

Comment:
Should there be a YANG file for this?

Reply:
That protogen generates from?

Reply:
There could be -- just hand creating this is probably better though since it will give the opportunity to add invalid proto fields (e.g., those that don't have the right annotations) in the future.